graph/load.go

```diff
@@ -68,7 +68,7 @@ func (s *TagStore) CmdLoad(job *engine.Job) engine.Status {
 
 		for imageName, tagMap := range repositories {
 			for tag, address := range tagMap {
-				if err := s.Set(imageName, tag, address, true); err != nil {
+				if err := s.SetLoad(imageName, tag, address, true, job.Stdout); err != nil {
```

Comment:
But does it mean that when it doesn't go through (such as needing `force`), we're gonna get both a warning and an error?